web/src/components/HowItWorks.js

```diff
@@ -62,26 +63,26 @@ const HowItWorks = () => {
           <div className="py-8">
             <dl className="grid grid-cols-12 gap-y-4 md:gap-8">
               <Feature
-                title="Simple config language"
+                title="Simple config"
                 url="/docs/general/language"
                 description="Declaratively describe high-level details of your app."
               />
 
               <Feature
-                title="Wasp CLI"
-                url="/docs/general/cli"
-                description="All the handy commands at your fingertips."
+                title="Goodbye boilerplate"
```

Comment:
Can we kick out this video? Learn more should either link to our docs, or to nothing else, this way we're adding links for the sake of fun, and it has SEO implications.